department-of-veterans-affairs/va.gov-cms
# VACMS-12378: updated styling for textfield counter warning and over

## Description

Closes #12377.
Closes #12378.

Change styling of textfield counter for a11y.

## Testing done
Tested locally. I tested this on both a Benefits Detail page and VAMC Detail page int he page Title and Meta Description fields. 

## Screenshots


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

- [ ] Update text color for character counter to always be `#212121`
- [ ] Update the border outline around the field to `#C58702` when between 5 and 0 characters away from the character limit
- [ ] Update the border outline around the field to be to `#B20D01` when the number of characters exceeds the character limit
Note: The hover state tends to hide the border, this is a separate issue. You may need to click off the field to see the border color change. 

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
